WMFComponents/Sources/WMFComponents/Extensions/UIViewController+NavigationBarUtils.swift

```diff
@@ -0,0 +1,333 @@
+// Common methods used for navigation bar display in the app
+import UIKit
+
```

Comment:
Optional: I think WMFNavigationBarConfiguring deserves to be in it's own file, rather than in a Utils file, same as WMFNavigationBarHiding.

Reply:
Done!